### ISSUE TITLE
[cloud_firestore] query if a field value is in some list

### ISSUE THREAD

**Issue by Habib23me** [FirebaseExtended/flutterfire#166]
I have been at this problem for a week now trying to find a way around with no success. 

The problem I am facing is that I have a collection called "questions" which contains documents with a field named "submittedBy" whose value is a string (which is a userId of the user that submitted the question).

I also have a users collection containing user documents. The documents contain a field called "following" which is a list of string (userIds that the user is following).

What I want to do is query the "questions" collection to find only questions submitted by userIds which are on my following list.

In other words, I want to check whether the value of the field "submittedBy" of each document is inside my list of following userIds.

Please let me know a way I could make this query or if not maybe a workaround to structure my data. 

Thanks in advance.

**Comment by creativecreatorormaybenot** [comment]
This should probably be a StackOverflow question and is most likely unrelated to Flutter because it is about Firestore database structuring in general.

**Comment by BondarenkoStas** [comment]
@Habib23me This is a question better suited for StackOverflow, since there doesn’t seem to be anything wrong with Flutter itself, but related to implementation. Closing, as this is not an issue report or feature request. See https://flutter.dev/community for support options.
If you disagree, feel free to comment and I will reopen the issue. Thank you for understanding.
